src/main/java/com/google/codeu/servlets/MapsServlet.java

```diff
@@ -0,0 +1,56 @@
+package com.google.codeu.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import java.io.IOException;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Returns Starbucks Locations as an array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}]
+ */
+@WebServlet("/starbucks-data")
+public class MapsServlet extends HttpServlet {
+
+  JsonArray starbucksLocationsArray;
+
+  @Override
+  public void init() {
+    starbucksLocationsArray = new JsonArray();
+    Gson gson = new Gson();
+    Scanner scanner =
```

Comment:
Let's move the scanning into a method. Usually I would prefer to abstract out large code into a more descriptive function since "init" doesn't really tell us all that is going on in it.

Maybe move the scanning out to a parseFile(String string) method?